settings.ini

```
@@ -4,7 +4,7 @@ lib_name = fasthtml
 version = 0.0.11
 min_python = 3.10
 license = apache2
-requirements = fastcore>=1.5.45 python-dateutil starlette oauthlib itsdangerous uvicorn httpx fastlite>=0.0.6 python-multipart
+requirements = fastcore>=1.5.45 python-dateutil starlette oauthlib itsdangerous uvicorn[standard] httpx fastlite>=0.0.6 python-multipart sqlite-utils
```

```suggestion
requirements = fastcore>=1.5.45 python-dateutil starlette oauthlib itsdangerous uvicorn[standard] httpx fastlite>=0.0.6 python-multipart
```

It shouldn't be necessary - I'm guessing one of your deps might need updating? We've switched to using sqlite-minutils now, which is a dep on fastlite.